project-base/storefront/components/Pages/Stores/StoresContent.tsx

```diff
@@ -21,40 +19,22 @@ type StoresContentProps = {
 export const StoresContent: FC<StoresContentProps> = ({ stores }) => {
     const { t } = useTranslation();
     const { defaultLocale } = useDomainConfig();
-    const [activeStoreIndex, setActiveStoreIndex] = useState<number>();
+    const [activeStoreId, setActiveStoreId] = useState<string>();
     const mappedStores = useMemo(() => mapConnectionEdges<TypeListedStoreFragment>(stores.edges), [stores.edges]);
 
-    const markers = useMemo(() => {
-        const validMarkers: Array<MapMarker> = [];
-        mappedStores?.forEach(({ locationLatitude, locationLongitude, slug }) => {
-            const marker = createMapMarker(locationLatitude, locationLongitude, slug);
-            if (marker) {
-                validMarkers.push(marker);
-            }
-        });
-
-        return validMarkers;
-    }, [mappedStores]);
-
-    const activeMarkerHandler = useCallback((markerId: string) => {
-        const newStoreIndex = mappedStores?.findIndex((store) => store.slug === markerId);
-
-        setActiveStoreIndex(activeStoreIndex !== newStoreIndex ? newStoreIndex : undefined);
+    const activeMarkerHandler = useCallback((id: string) => {
+        setActiveStoreId(activeStoreId !== id ? id : undefined);
```

Comment:
You might not need `useCallback` here. What benefit does it bring?

Reply:
Correct, unnecessary.